src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent, ReactNode } from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+export type ErrorMessageProps = {
+  id?: string;
+  className?: string;
+  visuallyHiddenText?: string;
```

Comment:
```suggestion
  visuallyHiddenText: string;
```

It's required since we pass in a default.